### PR TITLE
[SAM/CodeClean/Py] fix pylint and syntax error

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, fix pylint warning and syntax error.
- indent with 4 spaces
- function name rule: lowercase
- 2 blank lines between class and function

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Comment by dongju-chae** [comment]
Q. SAM measures a score of python codes as well?

**Comment by jaeyun-jung** [comment]
@dongju-chae Yes, I got a mail from se team, to fix & update python code :)

**Comment by again4you** [comment]
> Q. SAM measures a score of python codes as well?

Well, we got a report from SE guy. CQM score is higher than average but SAM isn't as below.
* https://analysishub.sec.samsung.net/service/analyses/757966

File: tests/gen24bBMP.py [hunk 1]
```diff
@@ -410,10 +405,11 @@ def gen_BMP_stream(filename_prefix, golden_filename, num_sink):
             else:
                 pixels[9] += [0, 0, 0]
 
+    string = []
     with open(golden_filename, 'wb') as file:
         for i in range(0, 10):
             saveBMP(filename_prefix + '_' + str(i) + '.bmp', pixels[i], 'RGB', 16, 16)
```
Comment (wooksong) [inline tests/gen24bBMP.py:411]: FYI , ```autopep8``` changes this line as follows:
```suggestion
            saveBMP(filename_prefix + '_' + str(i) +
                    '.bmp', pixels[i], 'RGB', 16, 16)
```

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM👍 Great thanks!

**Pull request merged**